### PR TITLE
Fresh images for Asset Pack Editor chapter

### PR DESCRIPTION
**Pull request by michaelgundlach**
This is to go along with the renaming of `Add File` to `Import File` etc in the Asset Pack Editor.

Reproduced all your images from the Asset Pack Editor chapter as closely as possible, using the Dark theme instead of Light as I noticed your more recent `.webp`s all used Dark theme.

### CODE REVIEW

**Comment by michaelgundlach** [comment]
I left all the old `.webp`s in the `images/` folder rather than deleting them.

**Comment by PhaserEditor2D** [comment]
> I left all the old `.webp`s in the `images/` folder rather than deleting them.

Yes, it is what I do.

Thanks for the fixes!

File: asset-pack-editor/import-from-blocks-view.rst [hunk 1]
```diff
@@ -38,5 +38,5 @@ Showing all files in the Blocks view
 
 As we mentioned before, the Blocks view only shows the files in the same folder (or sub-folder) as the |AssetPackFile|_. You can disable this restriction by checking the **Show All Files in Project** settings in the Inspector view:
 
-.. image:: ../images/asset-pack-show-all-files-in-blocks-20210529.webp
+.. image:: ../images/asset-pack-show-all-files-in-blocks-01012022.webp
```
Comment (PhaserEditor2D) [inline asset-pack-editor/import-from-blocks-view.rst:41]: I think this image is missing in the `images` folder.

Comment (PhaserEditor2D) [reply asset-pack-editor/import-from-blocks-view.rst:41]: This is the error:

```
/home/****/docs-repository/asset-pack-editor/import-from-blocks-view.rst:: WARNING: image file not readable: asset-pack-editor/../images/asset-pack-show-all-files-in-blocks-01012022.webp
```

**Comment by michaelgundlach** [comment]
Good catch - 2 of your images were from May 2021 and didn't need reproducing, but I only left one reference in place.  See PR #11.

**Pull request merged**